src/app/components/publiclists/teamlist/teamcard/teamcard.component.ts

```diff
@@ -13,6 +14,21 @@ import { Router, ActivatedRoute } from '@angular/router';
 })
 export class TeamcardComponent implements OnInit, OnChanges {
 
+  /**
+   * User Name
+   */
+  user = {username: ''};
+
+  /**
+   * Authentication Service subscription
+   */
+  authServiceSubscription: any;
+
+  /**
+   * Current Authentication state
+   */
+  authState: any;
```

Comment:
Do we need this variable? or you can keep this by removing [this](https://github.com/Cloud-CV/EvalAI-ngx/pull/288/files?file-filters%5B%5D=.html&file-filters%5B%5D=.ts&file-filters%5B%5D=.yml#diff-39040dd12941e102fae3dbad153f9262R20) one